src/components/VarCard.tsx

```diff
@@ -34,35 +34,38 @@ type VarCardProps = {
   deselectVariable: any; // method to call to remove a variable from the list
   hidden: boolean; // should the axis be hidden by default
   updateDimInfo: any; // method passed by the parent to update their copy of the variables dimension info
-  isSelected: boolean; // is this variable already selected
+  isSelected: Function; // method to check if this variable is selected in parent
```

Comment:
I didnt know you could use Function for a type. If this works we should go back and change any of the props that are set to <any> that are supposed to be methods.

Reply:
I through all the files and their props and made the update as suggested.